emamux.el

```diff
@@ -455,11 +455,21 @@ For helm completion use either `normal' or `helm' and turn on `helm-mode'."
   "Create new window by cd-ing to current directory.
 With prefix-arg, use '-a' option to insert the new window next to current index."
   (interactive)
-  (apply 'emamux:tmux-run-command nil
-         "new-window" (and current-prefix-arg '("-a")))
-  (let ((new-window-id (emamux:current-active-window-id))
-        (chdir-cmd (format " cd %s" default-directory)))
-    (emamux:send-keys chdir-cmd new-window-id)))
+  (let (cd-to ssh-to)
+    (if (file-remote-p default-directory)
+        (with-parsed-tramp-file-name
+            default-directory nil
+          (setq cd-to localname)
```

Comment:
I suppose that `host` should be checked. If it means localhost, don't launch `ssh` later. (NOTE: `tramp-local-host-regexp` is matched against `127.0.0.1`, `::1`, `localhost`, system name etc).